src/components/MultiHopTrade/components/RateGasRow.tsx

```diff
@@ -65,13 +68,15 @@ export const RateGasRow: FC<RateGasRowProps> = memo(
     swapperName,
     swapSource,
     networkFeeFiatUserCurrency,
+    deltaPercentage,
     onClick,
     noExpand,
-    isOpen: defaultIsOpen,
+    invertRate = false,
```

Comment:
turbo nitpick: either not optional or don't default to false, this effectively akin to required

Reply:
Changed in [2fc1511](https://github.com/shapeshift/web/pull/8970/commits/2fc15110e25204d7f59b309c7b574a6c6b83118d)